### PR TITLE
Add sample for M5Stack with BMP280 Sensor, AzureSDK and Certificates

### PR DESCRIPTION
**Pull request by ikivanc**
This sample application includes implementation of BMP280 sensor with M5Stack, pushes data through AzureSDK using certificates to Azure IoT Hub and save all data to a Azure Blob Storage.

## Description
This is a connected scenario of M5Stack, sensor, Azure with device certificates.

## Motivation and Context
Sample application includes a scenario: Every 3 seconds application reads temperature value from BMP280 sensor and turns on LED after reading the value. Then this temperature value is pushed to IoT Hub with device name and after every successful push LED will turn off. Data will be handled by IoT Hub message routing and will be pushed to blob storage to store.

## How Has This Been Tested?
* This has been on M5Stack using the latest preview of Nanoframework build 
  `nanoff --target ESP32_WROOM_32 -serialport COM6 --update --preview`
* Using Adafruit BMP280
* A certificated created manually

## Screenshots
image
image

## Types of changes
- [ ] Improvement (non-breaking change that improves a sample)
- [ ] Bug fix (fixes an issue with a current sample)
- [x] New Sample (adds a new sample)
- [ ] Config and build (change in the configuration and build system, has no impact on code or features)

## Checklist:
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

Signed-off-by: ikivanc

### CODE REVIEW

**Review by Ellerbach — APPROVED** [review]
Looks good to me all up, few comments.

File: samples/AzureSDK/AzureSDKSensorCertificate/Readme.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
+# Using Azure SDK with BMP280 on M5Stack with .NET nanoFramework
+
+M5Stack is a modular, stackable, and portable device which is powered with an ESP-32 core. And .NET nanoFramework image can be loaded in the board to enable the writing of managed code applications.
```
[the remark this answers is not in the document]
Comment (ikivanc) [reply samples/AzureSDK/AzureSDKSensorCertificate/Readme.md:3]: Thanks Laurent, I addressed this in the doc

File: samples/AzureSDK/AzureSDKSensorCertificate/Readme.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+* GPIO 18 to LED anode (longer, positive lead)
+* LED cathode (shorter, negative lead) to 330 Ω resistor (either end)
+* 330 Ω resistor (other end) to ground
+
+## Run the Solution
+
+Use your Azure Subscription for following steps:
+
+1. Make sure your [Pins](#-Pins) are correctly connected.
+2. Create an IoT Hub using your Azure Subscription and leave the authentication options to default (Symmetric Key, autogenerated). You'll use IoT Hub name in your code. You can use free tier of IoT Hub as well.
```
[the remark this answers is not in the document]
Comment (ikivanc) [reply samples/AzureSDK/AzureSDKSensorCertificate/Readme.md:48]: Yes, this was referring the previous implementation I did, thanks for pointing out.

File: samples/AzureSDK/AzureSDKSensorCertificate/Readme.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+    const string IotBrokerAddress = "<replace-with-your-iot-hub-name>.azure-devices.net";
+    ```
+
+6. Enter your WiFi Settings in the `Program.cs` file
+
+    ```csharp
+    const string  MySsid = "<replace-with-valid-ssid";
+    const string  MyPassword = "<replace-with-valid-password>";
+    ```
+
```
[the remark this answers is not in the document]
Comment (ikivanc) [reply samples/AzureSDK/AzureSDKSensorCertificate/Readme.md:73]: Added as a step into documentation.

File: samples/AzureSDK/AzureSDKSensorCertificate/Readme.md [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+### Building the sample
```
[the remark this answers is not in the document]
Comment (ikivanc) [reply samples/AzureSDK/AzureSDKSensorCertificate/Readme.md:96]: Fixed it.

File: samples/AzureSDK/AzureSDKSensorCertificate/NFSensorAzureIoTHub/app.config [hunk 5]
```diff
@@ -0,0 +1,47 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (josesimoes) [inline samples/AzureSDK/AzureSDKSensorCertificate/NFSensorAzureIoTHub/app.config:1]: we shouldn't have app.config in nano projects... Please remove

File: samples/AzureSDK/DpsSampleApp/app.config [hunk 6]
```diff
@@ -0,0 +1,27 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (josesimoes) [inline samples/AzureSDK/DpsSampleApp/app.config:1]: Same here...

**Review by josesimoes — CHANGES_REQUESTED** [review]
Just remove the app configs please and we're good to merge! 🙂

File: samples/AzureSDK/AzureSDKSensorCertificate/NFSensorAzureIoTHub/app.config (hunk 5 above)
Comment (ikivanc) [reply samples/AzureSDK/AzureSDKSensorCertificate/NFSensorAzureIoTHub/app.config]: Thanks Jose, I removed app.config files

File: samples/AzureSDK/DpsSampleApp/app.config (hunk 6 above)
Comment (ikivanc) [reply samples/AzureSDK/DpsSampleApp/app.config]: this is removed as well

**Review by josesimoes — APPROVED** [review]
Perfect! Thanks for the new sample. 👍

**Pull request merged**